src/main/java/core/basesyntax/processing/RawDataProcessor.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.processing;
+
+import core.basesyntax.enums.Operation;
+import core.basesyntax.handler.TransactionHandler;
+import core.basesyntax.handler.impl.BalanceTransactionHandler;
+import core.basesyntax.handler.impl.PurchaseTransactionHandler;
+import core.basesyntax.handler.impl.ReturnTransactionHandler;
+import core.basesyntax.handler.impl.SupplyTransactionHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RawDataProcessor {
+    private final List<List<String>> rawData;
```

Comment:
```suggestion
```
if something can be a local variable, it should be a local variable. we should make variable scope as minimal as possible.  Keeping variable scope as minimal as possible helps prevent unintended side effects and makes the code easier to read and maintain. Local variables are generally preferred because they limit the visibility of variables to the smallest context necessary, reducing the chances of naming conflicts and improving overall code clarity.